src/main.rs

```diff
@@ -1,10 +1,14 @@
 #![allow(unused_features)]
+#![feature(std_misc)]
 #![feature(static_mutex)]
 #![feature(box_syntax)]
+#![feature(libc)]
```

Comment:
Ya, you're using rust's builtin libc.  I'm just going off of the warning on http://doc.rust-lang.org/nightly/libc/index.html.